hoverctl/wrapper/hoverfly.go

```diff
@@ -188,8 +189,45 @@ func SetMiddleware(target Target, binary, script, remote string) (v2.MiddlewareV
 	return apiResponse, nil
 }
 
+func GetLogs(target Target, format string) ([]string, error) {
+	headers := map[string]string{
+		"Content-Type": "text/plain",
+	}
+
+	if format == "json" {
+		headers["Content-Type"] = "application/json"
+	}
+
+	response, err := doRequest(target, "GET", v2ApiLogs, "", headers)
+	if err != nil {
+		return []string{}, err
+	}
+
+	defer response.Body.Close()
+
+	responseBody, _ := ioutil.ReadAll(response.Body)
```

Comment:
I have updated this now so if the format is JSON, it will unmarshal the JSON and then the individual JSON logs, avoiding any potentially dodgey string manipulation. If the format isn't JSON, it'll assume its plaintext, which is the default in Hoverctl and still split that by line.